src/infra/cache/multi-level-memoizee.ts

```diff
@@ -105,6 +110,7 @@ export function multiCacheMemoizee<
 
 	const multiCacheOpts: Parameters<typeof createMultiLevelStore>[1] = {
 		default: { ...convertToMultiStoreOpts(opts), isCacheableValue: () => true },
+		...(opts.maxAge === 0 && { local: false }),
```

Comment:
Actually memoizee caches indefinitely w/ `maxAge: 0`, so this would be a discrepancy between memoizee & multi-level-memoizee :-1: